modules/create_timepoint/test/create_timepointTest.php

```diff
@@ -117,6 +119,12 @@ function testCreateTimepointErrorEmptySubproject()
         $this->safeGet(
             $this->url . "/create_timepoint/?candID=900000&identifier=900000"
         );
+        $this->webDriver->wait(10, 1000)->until(
```

Comment:
Without this code the ajax request has not completed and the assertion fails (since the upgrade).

Reply:
$this->safeFindElement(WebDriverBy:: ..... ) will be better. 
This function has solved the ajax call issue.

Reply:
Thanks @kongtiaowang , it works like a charm!

Reply:
The change introduced a new failure. I'm investigating.